circle.yml

```diff
@@ -8,9 +8,16 @@ machine:
     CODECLIMATE_DOCKER_REGISTRY_USERNAME: circleci
     CODECLIMATE_DOCKER_REGISTRY_EMAIL: ops@codeclimate.com
 
-test:
+dependencies:
   override:
     - docker build -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
+    - npm install mocha
```

Comment:
These should be listed in the package.json file and you should just run `npm install`. I can help you with that tomorrow!